### DISCUSSION TITLE
Missing Project Roadmap

### DISCUSSION THREAD

**Discussion by andr229k** [EMATech/midiexplorer#10] [General]
I found this project by a Google search and is very interested, especially around the “Support proprietary formats”. I’m only able to find a “To-do list”, but not a planned “roadmap” for when the different features are planned to be rolled out.

**Comment by rdoursenaud** [comment]
Hi and thank you for your interest in my humble program.

This is something I work on in my spare time for my own needs and share here for free.
The TODO list is a list of topics I want to explore sometime and would like to ultimately feature if it makes sense.
For this reason, there is no roadmap.
I don't make any commitment to provide anything further what's available here and release what I'm working on regularly.

Could you please elaborate on which specific feature you'd be interested-in regarding proprietary formats?

**Comment by andr229k** [comment]
I do repairs of old music instruments and I find your idea is great, as there is not really a “bridge” between the different devices' content, when it comes to SMF for those who are not directly SMF compatible.

**My interest in test your software is around:**
Alesis, Ensoniq, Kawai, Korg, Roland and Yamaha

Something that is maybe worth to consider, is the end-user perspective, something that is a “block” for basic/non-technical end-users is how this platform is build around installing Python with additionally installation PipX instead of having it compiled into 1 x .exe file.
